san/sanbase_graphql_helper.py

```diff
@@ -6,11 +6,7 @@
 _DEFAULT_SOURCE = 'TELEGRAM'
 _DEFAULT_SEARCH_TEXT = ''
 
-_QUERY_MAPPING = {
-    'daily_active_addresses': {
```

Comment:
why remove `daily_active_addresses` ?

Reply:
Because it currently uses the daily_active_addresses from getMetric and not the dailyActiveAddresses API

Reply:
It returns ['datetime', 'activeAddresses']  while the getMetric on returns ['datetime', 'value']. That will break the code of those currently using it. 
Isn't it the same case with `network_growth`, `dev_activity` , etc but they are not removed ?
Also didn't we have a message that we will deprecated these that are not using `getMetric` in V. 0.9.0 ?

Reply:
The PR just got a little streched out and I forgot about it, the main reason why I removed it is because there is no daily_active_addresses function in the first place, maybe we've agreed to deprecate the old API for the getMetric some time back, don't really know why it isn't here now. Another reason is that the 'get' function currently favours the getMetric version more over the old API one, the "switch" that we've made in this PR will change that and will break stuff anyways, but you're right about the case with network_growth and dev_activity - we should decide on whether we remove these same-named old APIs altogether or leave all of them be, I think we should remove them all personally

Reply:
Ok, got it. In that case we should find all queries like `network_growth`, etc that are in both getMetric and without and remove them also. Because currnetly if you call `network_growth`, you get `value` field. After this PR you'll get `newAddresses` - so that breaks existing code. If we remove also `network growth` - it will go to getMetric and you'll get `value` again.
So we should remove all that are also in getMetric.

Reply:
And also update the README that it returns `value` cause I guess we've missed that previous time.

Reply:
Done, also added some more metrics to the to-be-deprecated metrics list